src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -418,9 +426,39 @@ func (c controller) postInstallConfigs(ctx context.Context) error {
 		if err = c.updatePendingOLMOperators(); err != nil {
 			return errors.Errorf("Timeout while waiting for some of the operators and not able to update its state")
 		}
-		c.log.Warnf("Timeout while waiting for OLM operators be installed")
+		c.log.WithError(err).Warnf("Timeout while waiting for OLM operators be installed")
+		// Don't return the error here so the installation is marked as sucessful.
+		// We only log the error, and update the state of the monitored operators in assisted-service API.
+	}
+
+	return nil
+}
+
+func (c controller) applyPostInstallManifests() error {
+	ctx := utils.GenerateRequestContext()
+	tempDir, err := ioutil.TempDir("", "controller-custom-manifests-")
+	if err != nil {
+		return err
+	}
+
+	customManifestPath := path.Join(tempDir, customManifestsFile)
+	if err := c.ic.DownloadFile(ctx, customManifestsFile, customManifestPath); err != nil {
+		return err
+	}
+
+	kubeconfigName, err := c.downloadKubeconfigNoingress(ctx, tempDir)
```

Comment:
will we retry in case of failure?

Reply:
We retry only on `oc apply` failures, do you think we should retry in all the errors?

Reply:
i think we should cause on any failure we will fail installation.

Reply:
for example service can be not reachable for a minute and we will fail the installation cause of it?

Reply:
Right, will re-work.